pkg/rke2/kp.go

```diff
@@ -36,7 +35,8 @@ func setKubeProxyDisabled(clx *cli.Context, cfg *cmds.Server) func(context.Conte
 			if _, err := hc.Helm().V1().HelmChartConfig().Get(metav1.NamespaceSystem, kubeProxyChart, metav1.GetOptions{}); err != nil {
 				if apierrors.IsNotFound(err) {
 					logrus.Infof("%[1]s HelmChartConfig not found, disabling %[1]s", kubeProxyChart)
-					clx.Set("disable", kubeProxyChart)
+					args.Skips[kubeProxyChart] = false
```

Comment:
We **do** want to skip and disable these.

Reply:
ah yes